### ISSUE TITLE
Configure the generated Secrets class accessibility

### ISSUE THREAD

**Issue by chowarth** [dansiegel/Mobile.BuildTools#172]
### SUMMARY

Add the ability to configure the `Secrets` class accessibility.

### DETAILS

Currently the generated `Secrets` class is always given `internal` accessibility. There may be scenarios where you want the secrets class to be part of a common project that is shared by multiple other projects, so adding ability to configure this via the `buildtools.json` would be a great addition. Supported values should be the standard C# accessibility levels e.g.

- public
- protected (?)
- private (?)
- internal (default if not specified)

While there is a workaround, by using the `InternalsVisibleTo` attribute, this is a little cumbersome.

```json
"projectSecrets":
{
    "Project.Name":
    {
        "disable":false,
        "delimiter":";",
        "prefix":"BuildTools_",
        "className":"Secrets",
        "classAccessibility": "public", // New property
        "namespace":"Helpers",
        "properties":[
            {
                "name":"SomeSecret",
                "type":"String",
                "isArray":false
            }
        ]
    }
}
```

### PLATFORMS

- [x] Android
- [x] iOS
- [x] UWP

**Comment by dansiegel** [comment]
this is a great enhancement. I've added it to the 2.0 milestone

**Comment by dansiegel** [comment]
I should note that this will only support internal or public. A protected/private class wouldn't make any sense in this context.
